bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/scriptengine/InvocationInterceptingScriptEngineWithInvocable.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
```

Comment:
All headers probably need to be updated to 2021:

```suggestion
 * Copyright (c) 2010-2021 Contributors to the openHAB project
```